namespace-resources/main.tf

```diff
@@ -0,0 +1,69 @@
+locals {
+  cluster = "${var.cluster}.k8s.integration.dsd.io"
+}
+
+data "template_file" "namespace" {
+  template = "${file("${path.module}/00-namespace.yaml")}"
+
+  vars {
+    namespace = "${var.namespace}"
+  }
+}
+
+resource "local_file" "00-namespace" {
+  content  = "${data.template_file.namespace.rendered}"
+  filename = "../namespaces/${local.cluster}/${var.namespace}/00-namespace.yaml"
+}
+
+data "template_file" "rbac" {
+  template = "${file("${path.module}/01-rbac.yaml")}"
+
+  vars {
+    namespace   = "${var.namespace}"
+    github_team = "${var.github_team}"
+  }
+}
+
+resource "local_file" "01-rbac" {
+  content  = "${data.template_file.rbac.rendered}"
+  filename = "../namespaces/${local.cluster}/${var.namespace}/01-rbac.yaml"
+}
+
+data "template_file" "limitrange" {
+  template = "${file("${path.module}/02-limitrange.yaml")}"
+
+  vars {
+    namespace = "${var.namespace}"
+  }
+}
+
+resource "local_file" "02-limitrange" {
+  content  = "${data.template_file.limitrange.rendered}"
+  filename = "../namespaces/${local.cluster}/${var.namespace}/02-limitrange.yaml"
+}
+
+data "template_file" "resourcequota" {
+  template = "${file("${path.module}/03-resourcequota.yaml")}"
+
+  vars {
+    namespace = "${var.namespace}"
+  }
+}
+
+resource "local_file" "03-resourcequota" {
+  content  = "${data.template_file.resourcequota.rendered}"
+  filename = "../namespaces/${local.cluster}/${var.namespace}/03-resourcequota.yaml"
```

Comment:
I wonder if it would be better to prepend these paths with `${path.module}` as well.

Reply:
discussed.